keyboards/ymd96/ymd96.h

```diff
@@ -76,5 +76,22 @@ K411, K410, K409, K408, K404, K405, K406, K407, K403, K402, K401, K400  \
 {      K400,       K401,       K402,       K403,       K404,       K405,       K406,       K407,       K408,       K409,       K410,       K411 },  \
 }
 
+#define LAYOUT_iso( \
```

Comment:
This appears to support the "Fullsize ISO" layout? 

If so, then you may want to use `LAYOUT_fullsize_iso` here, instead: 

https://github.com/qmk/qmk_firmware/blob/master/layouts/default/fullsize_iso/default_fullsize_iso/keymap.c